paper/sections/materials.tex

```diff
@@ -83,10 +86,9 @@ \subsection{Model Flexibility}
 \item the sorting and balancing of samples within the dataset,
 \item the selected SVM hyperparameters, gamma and cost,
 \item the uniformity (or possible lack of) SVM hyperparameters across binary classifiers,
-\item the aggregation technique applied to binary classifiers,
-\item the prediction method for the final labels, and\\
+\item the aggregation technique applied to binary classifiers, and \\
 \Hamid{for evalueation metrics comment below:}
-{\item the evaluation metrics averaging technique (macro vs micro).}
+{\item the averaging technique for calculation of final performance metrics of a model (macro vs micro).}
```

Comment:
@HamidHeyde please revert the changes on the current lines 89 and 91. The changes break the order of the list and the downstream paragraph. The bullets aren't the same -- one is *when* the aggregation takes place (micro vs macro), the other is *how* the aggregation is done (max prob, weighted-mean, or unweighted mean).

Reply:
@gkiar: You mean i need to change the order? so, micro vs macro => 5 and aggregation technique => 6  or revert it back to what it was before?

Reply:
I mean to exactly as they were written before this PR, yes. Thanks!

Reply:
sure, no problem for me. 
Sorry @gkiar, I'm a bit confused, this way of communication sometimes gives the wrong idea. 
Correct me if i'm wrong, but before it was (5)=>aggregation technique and (6)=>  final label prediction method? and since these 2  (to my understanding) are referring to the same things, then wasn't that why Tristan left that comment (micro vs macro) and i had to add a Bullet Point for that? 

So, instead of aggregation technique(=>5) shouldn't we say, performance metrics averaging technique? since you have already mentioned that point in the "2.3.3-Performance Evaluation" (addr:5)

So, if that is not the case then what i should i write in regards to Tristan's comment? Should i add a new bullet point? or it has been already taken care of?

Reply:
Hey @HamidHeyde - I agree, looking at partial diffs does indeed get hard to track and follow!

I believe that @glatard 's comment was covered in the paragraph below and under bullet 5, as we said, because we explicitly link it to the macro/micro approach, there. I'll add a note to suggest that we perhaps adjust the wording of the 5th bullet to make that more clear, but the sequence and content should otherwise be unchanged. Thanks for reverting and the other changes!

Reply:
Thanks @gkiar